### PR TITLE
Implement resources screen with bootsrap ui

### PR DESCRIPTION
**Pull request by fpagnoux**
Make it work on Firefox on IE

Praise @Flightan :bow: :bow: :bow:

### CODE REVIEW

**Comment by fpagnoux** [comment]
image

File: app/views/partials/foyer/ressources/types.html [hunk 1]
```diff
@@ -5,23 +5,26 @@
...
+            <div class="capture-rows">
+              <button
+                type="button"
+                class="btn btn-sm btn-selection btn-default"
+                ng-repeat="ressourceType in ressourceTypesByCategories[category.id]"
+                btn-checkbox
+                ng-model="individuVM.selectedRessourceTypes[ressourceType.id]">
+                {{ ressourceType.label }}
+              </button>
+            </div>  </accordion-group>
```
Comment (MattiSG) [inline app/views/partials/foyer/ressources/types.html]: Line break.

File: app/views/partials/foyer/ressources/types.html [hunk 2]
```diff
@@ -5,23 +5,26 @@
   vous pourrez ensuite saisir les montants.
 </p>
 
+
+
```
Comment (MattiSG) [inline app/views/partials/foyer/ressources/types.html]: Why these additional line breaks?

**Comment by fpagnoux** [comment]
Thanks for your feedback ! Is this good now ?

**Comment by MattiSG** [comment]
Should be, I just need to test it manually…

**Comment by MattiSG** [comment]
I'm really sorry, but I don't think this behaviour is proper. Only one item can be open at the time, which means I have no clue what I have already clicked on. The current behaviour to keep open tabs open is perhaps not the best as it allows one to clutter the screen, but this one is not acceptable: if I toggle a resource on, it is pretty hard to toggle it back off: where was it? What have I already selected?

One workaround could perhaps be to add the number of toggled resource types in the header (not great, but a workaround).

**Comment by MattiSG** [comment]
(FYI I have already looked at the [Bootstrap doc](http://getbootstrap.com/javascript/index.html#collapse-options) and there doesn't seem to be a way to allow multiple selections on an accordion)

**Comment by fpagnoux** [comment]
There must be a way to dis-activate the "one at a time" behavior. Looking at it right now.

**Comment by fpagnoux** [comment]
It seems to be possible to avoid this behavior. I'm on it.
https://angular-ui.github.io/bootstrap/#/accordion

**Comment by fpagnoux** [comment]
That should work. @MattiSG, is it ok on your browser too ?

**Comment by MattiSG** [comment]
GTM!!

**Pull request merged**